From 09193082759c2fb67bc4c11dc6e6672855f8dba0 Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Thu, 18 Oct 2018 21:13:10 +0100
Subject: [PATCH 5/5] CMake: Fix windres / .rc not including programs/windres

---
 build/cmake/lib/CMakeLists.txt | 17 +++++++++++++++--
 1 file changed, 15 insertions(+), 2 deletions(-)

diff --git a/build/cmake/lib/CMakeLists.txt b/build/cmake/lib/CMakeLists.txt
index ffc196dd..c69ef0f0 100644
--- a/build/cmake/lib/CMakeLists.txt
+++ b/build/cmake/lib/CMakeLists.txt
@@ -102,10 +102,12 @@ IF (ZSTD_LEGACY_SUPPORT)
             ${LIBRARY_LEGACY_DIR}/zstd_v07.h)
 ENDIF (ZSTD_LEGACY_SUPPORT)
 
-IF (MSVC)
+IF (WIN32)
     SET(MSVC_RESOURCE_DIR ${ZSTD_SOURCE_DIR}/build/VS2010/libzstd-dll)
     SET(PlatformDependResources ${MSVC_RESOURCE_DIR}/libzstd-dll.rc)
-ENDIF (MSVC)
+    SET_SOURCE_FILES_PROPERTIES(${MSVC_RESOURCE_DIR}/libzstd-dll.rc PROPERTIES COMPILE_FLAGS -I${ZSTD_SOURCE_DIR}/programs/windres)
+    SET_SOURCE_FILES_PROPERTIES(${MSVC_RESOURCE_DIR}/zstd.rc PROPERTIES COMPILE_FLAGS -I${ZSTD_SOURCE_DIR}/programs/windres)
+ENDIF (WIN32)
 
 # Split project to static and shared libraries build
 IF (ZSTD_BUILD_SHARED)
@@ -127,6 +129,17 @@ IF (ZSTD_BUILD_STATIC)
     ENDIF ()
 ENDIF (ZSTD_BUILD_STATIC)
 
+IF (WIN32)
+    IF (ZSTD_BUILD_SHARED)
+      TARGET_INCLUDE_DIRECTORIES(libzstd_shared -I${ZSTD_SOURCE_DIR}/programs/windres)
+      SET_TARGET_PROPERTIES(libzstd_shared PROPERTIES PREFIX "lib")
+    ENDIF (ZSTD_BUILD_SHARED)
+    IF (ZSTD_BUILD_STATIC)
+       TARGET_INCLUDE_DIRECTORIES(libzstd_static -I${ZSTD_SOURCE_DIR}/programs/windres)
+       SET_TARGET_PROPERTIES(libzstd_static PROPERTIES PREFIX "lib")
+    ENDIF (ZSTD_BUILD_STATIC)
+ENDIF (WIN32)
+
 # Add specific compile definitions for MSVC project
 IF (MSVC)
     IF (ZSTD_BUILD_SHARED)
-- 
2.19.1

